contracts/voting-verifier/src/msg.rs

```diff
@@ -10,17 +10,25 @@ use connection_router_api::{ChainName, CrossChainId, Message};
 
 #[cw_serde]
 pub struct InstantiateMsg {
+    /// Address that can call all messages of unrestricted, admin and governance permission level, like UpdateVotingThreshold.
```

Comment:
There's no admin for the voting verifier, so maybe you should remove the mention of admin. What do you think?